### DISCUSSION TITLE
Colorblind-friendly online/offline status dot

### DISCUSSION THREAD

**Discussion by TtuxX** [rustdesk/rustdesk#9544] [General]
Hi there @rustdesk and thanks for this awesome and open-source app :)

My problem: I'm systematically having a hard time differentiating which client is online and which is not in the different Rustdesk clients.

To differentiate them at the moment (and for you to understand the feeling) I always have to try to figure out first which dot color is more "bright" and which is more "dark", but it's not flawless and requires concentration.

I'd love indicator dots to be slightly larger and to have colors like grey for offline, and flashy bright green for online ?

Maybe colors like these (do not take shapes into account):

Online:
![image](https://github.com/user-attachments/assets/582e0f63-e673-4195-b014-5f3dae0285b7)

Offline (or slightly darker than this ?):
![image](https://github.com/user-attachments/assets/5e99a2ba-2152-4822-b952-068dfdf36bbc)

Many thanks in advance for considering this improvement request :)

**Comment by zeeter82** [comment]
I would also love a feature like this.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/11930 has a good suggestion

> Icons (checkmark vs X)

**Reply by rustdesk** [reply]
But it seems the dot is too small to contain a visible checkmark or x.

**Reply by Shablone** [reply]
Lets make it bigger then.

How about these commercial free SVGs from uxwing.com?
![grafik](https://github.com/user-attachments/assets/e58d541e-645f-4f88-8763-5a48e0540315)
[online svg](https://uxwing.com/wp-content/themes/uxwing/download/communication-chat-call/correct-icon.svg)
[offline svg](https://uxwing.com/wp-content/themes/uxwing/download/checkmark-cross/wrong-icon.svg)

Or even rather SVGsfor connected/disconnected. But I couldn't find proper ones.

**Reply by rustdesk** [reply]
Looks not good looking for me.

**Reply by rustdesk** [reply]
Do you also have trouble with below color dots from Apple? Also green and orange.

![image](https://github.com/user-attachments/assets/6e91b912-fccd-4b97-aca9-991ab43174f4)

**Reply by zeeter82** [reply]
Those might be a bit better for me, but still not passable in my opinion.  Whether it's an icon or a different color, it needs to be 100% without a doubt.

**Reply by Shablone** [reply]
I agree.
This problem is well studied. If you want to rely solely on color as long as there is not better alternative, I propose to use recommended color palettes, see:
[Scheme](https://www.nceas.ucsb.edu/sites/default/files/2022-06/Colorblind%20Safe%20Color%20Schemes.pdf)


Also how do you feel about these? Could be also colorized:
[offline](https://uxwing.com/offline-icon/)
[online](https://uxwing.com/wifi-line-icon/)

**Reply by Shablone** [reply]
Another proposal: Use greyed out text for the IDs of offline clients

**Reply by rustdesk** [reply]
So below ok for you.

![image](https://github.com/user-attachments/assets/170d9c60-dc67-41a7-8d93-0b0c0c59fe25)

**Reply by rustdesk** [reply]
You will have trouble if I choose two colors on different lines?

![image](https://github.com/user-attachments/assets/fb012f3a-3664-4164-9b8e-ed49f1dbc75f)

**Reply by Shablone** [reply]
Yes, I just checked it on the real thing, and for me it would be fine.
How about @TtuxX , @zeeter82?

You need to choose from a single palette though.

**Reply by zeeter82** [reply]
Seeing those large like that would prob work.  Smaller dots using those colors may still be difficult.

**Reply by toFrankie22** [reply]
Very true! I would love to see this change too. If I may suggest a check mark or a stop sign. But if we need to keep it super simple I would be even ok with a + or - sign. (+ for available and - for unavailable). Or anything that would not require to identify colors. Thank you!

**Reply by TtuxX** [reply]
@Shablone Thanks for asking and notification.
I see a bit the color difference suggested by @rustdesk and can understand which is active, but don't feel comfortable and cannot say which color it is as it is not so obvious.
That's why I initially suggested a clear visible "positive" green when connection is OK, and a light (or could be darker) grey "inactive" color.
That way, rather than having to differentiate between red and green, you would differenciate between "this one is alive/active", and the rest.
But happy to discuss as I understand it might not be the most intuitive as people are used to green/red.
@rustdesk And I completely agree with @zeeter82, the size of the dot changes the difficulty of perception, hence (for me) the oportance of making obvious the "active" items before anything

**Reply by eXtremeWarhead** [reply]
something new? it's still hard to see the difference in a blink of an eye...

**Reply by nicky-chris** [reply]
Hello.
Okay, I started a new thread about this and got redirected here. That's normal.

Personally, I'm also very embarrassed because I'm colorblind.

**My suggestion for improvement: Or perhaps to be able to sort only those who are online?**

Thanks to the developers
